api/controllers/UserController.js

```diff
@@ -19,6 +19,7 @@ async function getActiveMentors(req, res, next) {
 				activeMentors.push(user);
 			}
 		}
+		console.log(activeMentors);
```

Comment:
Remove console.logs. 

You should look over your PR before having people review it to catch simple things like these. It makes your code get accepted faster.